queries.sql

```diff
@@ -8,3 +9,45 @@ SELECT NAME, ESCAPE_ATTEMPTS FROM animals WHERE WEIGHT_KG>10.5;
 SELECT * FROM animals WHERE NEUTERED=true;
 SELECT * FROM animals WHERE NAME!='Gabumon';
 SELECT * FROM animals WHERE WEIGHT_KG BETWEEN 10.4 AND 17.3;
+
+/* Task Two */
+
+BEGIN;
+
+UPDATE animals SET SPECIES='unspecified';
+
+ROLLBACK;
+
+BEGIN;
+
+UPDATE animals SET SPECIES='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET SPECIES='pokemon' WHERE species IS NULL;
+
+COMMIT;
```

Comment:
- [ ] Similarly, as specified in the project requirement, you should provide a query that verifies the change before and after the `commit` as shown below 👇🏾 

```sql  
 BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit
```